### PR TITLE
[BD-46] docs: fixed issue feedback error

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

Issue: https://github.com/openedx/paragon/issues/2343

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [ ] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [ ] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [ ] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: src/Figure/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ notes: |
 
 <p className="lead">
   This is a pass through component from React-Bootstrap.<br/>
-  <a href="https://react-bootstrap.github.io/components/figures/" target="_blank" rel="noopener noreferrer">
+  <a href="https://react-bootstrap.github.io/docs/components/figures" target="_blank" rel="noopener noreferrer">
```
Comment (PKulkoRaccoonGang) [inline src/Figure/README.md:17]: Updated the link because the old link led to a nonexistent page

File: www/src/components/LeaveFeedback.tsx [hunk 2]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav } from '~paragon-react';
+import { useLocation } from '@reach/router';
 
 function LeaveFeedback(props) {
-  const CURRENT_URL = typeof window !== 'undefined' ? window.location.pathname : '';
-  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL})&amp;labels=docs&template=feedback_template.md`;
+  const CURRENT_URL = useLocation();
+  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL.pathname})&amp;labels=docs&template=feedback_template.md`;
```
Comment (monteri) [inline www/src/components/LeaveFeedback.tsx:8]: Maybe it is good idea to put it outside the component body?

File: www/src/components/LeaveFeedback.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Nav } from '~paragon-react';
+import { useLocation } from '@reach/router';
 
 function LeaveFeedback(props) {
-  const CURRENT_URL = typeof window !== 'undefined' ? window.location.pathname : '';
-  const FEEDBACK_URL = `https://github.com/openedx/paragon/issues/new?title=%5Bparagon-openedx.netlify.app%5D%20Feedback%20(on%20${CURRENT_URL})&amp;labels=docs&template=feedback_template.md`;
+  const CURRENT_URL = useLocation();
```
Comment (monteri) [inline www/src/components/LeaveFeedback.tsx:7]: I think here it is not a constant variable anymore. It means we can rename it to `const location = useLocation()`.

File: www/src/components/LeaveFeedback.tsx (hunk 2 above)
Comment (PKulkoRaccoonGang) [reply www/src/components/LeaveFeedback.tsx:8]: Thanks for the proposal!
The value of the `CURRENT_URL` variable is React Hook. React Hooks must be called in a React function component or a custom React Hook function.

File: www/src/components/LeaveFeedback.tsx (hunk 3 above)
Comment (PKulkoRaccoonGang) [reply www/src/components/LeaveFeedback.tsx:7]: Thanks, it's make sense 👍

**Review by adamstankiewicz — APPROVED** [review]
LGTM!

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2359&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-06-06+13:59&entry.1917517419=2023-06-08+11:53&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**